core/src/main/java/io/camunda/migrator/config/mybatis/SchemaShutdownCleaner.java

```diff
@@ -45,15 +49,23 @@ public class SchemaShutdownCleaner {
   @Autowired
   protected DbClient dbClient;
 
+  @Autowired
+  protected Environment environment;
+
   @PreDestroy
   public void cleanUp() {
```

Comment:
🙃 For logs we use `XXXLogs` classes in most places, so you can move them out from this class, up to you.